README.rst

```diff
@@ -49,7 +49,7 @@ scikit-learn is tested to work under Python 2.6, Python 2.7, and Python 3.4.
 `six <http://pythonhosted.org/six/>`_). It should also work with Python 3.3.
 
 The required dependencies to build the software are NumPy >= 1.6.1,
-SciPy >= 0.9 and a working C/C++ compiler.
+SciPy >= 0.9, Cython >=0.23 and a working C/C++ compiler. 
```

Comment:
That's only for a development version: you don't need cython to build from a released version. This should be made clear. Right now the formulation is unclear.